File: app/src/main/java/net/mieczkowski/yelpbusinessexample/models/auth/YelpAuth.java
package net.mieczkowski.yelpbusinessexample.models.auth;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.raizlabs.android.dbflow.annotation.Column;
import com.raizlabs.android.dbflow.annotation.Table;

import net.mieczkowski.yelpbusinessexample.database.MyDatabase;
import net.mieczkowski.yelpbusinessexample.database.PrimaryKeyModel;

/**
 * Created by Josh Mieczkowski on 10/19/2017.
 */

@Table(database = MyDatabase.class)
public class YelpAuth extends PrimaryKeyModel {

    @Column
    @JsonProperty("access_token")
    String accessToken;

    @Column
    @JsonProperty("token_type")
    String tokenType;

    public YelpAuth() {
    }

    public String getAccessToken() {
        return accessToken;
    }

    public void setAccessToken(String accessToken) {
        this.accessToken = accessToken;
    }

    public String getTokenType() {
        return tokenType;
    }

    public void setTokenType(String tokenType) {
        this.tokenType = tokenType;
    }


}

File: dal/src/main/java/net/mieczkowski/dal/services/authService/models/YelpAuth.kt
package net.mieczkowski.dal.services.authService.models

import com.fasterxml.jackson.annotation.JsonProperty
import com.raizlabs.android.dbflow.annotation.Column
import com.raizlabs.android.dbflow.annotation.Table
import net.mieczkowski.dal.cache.LocalDatabase
import net.mieczkowski.dal.cache.PrimaryKey

/**
 * Created by Josh Mieczkowski on 9/11/2018.
 */
@Table(database = LocalDatabase::class)
class YelpAuth: PrimaryKey() {

    @Column
    @JsonProperty("access_token")
    lateinit var accessToken: String

    @Column
    @JsonProperty("token_type")
    lateinit var tokenType: String
}